### ISSUE TITLE
KiTTY 0.74.4.9: `Can't shim 'kageant.exe': File doesn't exist`

### ISSUE THREAD

**Issue by redactedscribe** [lukesampson/scoop-extras#6059]
When installing: `Can't shim 'kageant.exe': File doesn't exist.`

```
Installing 'kitty' (0.74.4.9) [64bit]
Loading kitty-bin-0.74.4.9.zip from cache.
Checking hash of kitty-bin-0.74.4.9.zip ... ok.
Extracting kitty-bin-0.74.4.9.zip ... done.
Running pre-install script...
...
Creating shim for 'genpass'.
Creating shim for 'kageant'.
Can't shim 'kageant.exe': File doesn't exist.
```

```
These app manifests have been removed:
    kitty
These apps failed to install:
    kitty
```

KiTTY v0.74.4.9

**Comment by jrhawley** [comment]
I've found this error too, but it seems to be an issue with the [latest release of KiTTY itself](https://github.com/cyd01/KiTTY/releases/tag/v0.74.4.9):

> Unfortunately kageant can't be present in this release.
> Please use the embedded version of the agent with the command kitty.exe -runagent.

**Comment by redactedscribe** [comment]
Just don't uninstall it as it can't be reinstalled (unless a copy is still in your Scoop cache perhaps).

**Comment by jrhawley** [comment]
You can revert back to the latest version, that seems to be working for me, 

```powershell
scoop uninstall kitty
scoop install kitty@0.74.4.8
```

As long as you don't update KiTTY, that should continue to work until the next release is out.

**Comment by quantuumsnot** [comment]
I suggest you to backup your ssh keys. Now I'm having an issue restoring them because failed kitty removed hard-links for key directory. 40+ machines are not SSHable for the next 30-60mins

**Comment by redactedscribe** [comment]
A new version `0.74.4.10` was made available and installs, but with the message:
```
Creating shortcut for KiTTY\KiTTY Authentication Agent (kageant.exe) failed: Couldn't find <...>\scoop\apps\kitty\current\kageant.exe
```

**Comment by naderi** [comment]
This is the developer's comment for version 0.74.4.9 ([Release v0.74.4.9](https://github.com/cyd01/KiTTY/releases/tag/v0.74.4.9)):
"Unfortunately kageant can't be present in this release.
Please use the embedded version of the agent with the command kitty.exe -runagent"

@lukesampson Maybe the manifest should be updatetd.

**Comment by redactedscribe** [comment]
```
Creating shortcut for KiTTY\KiTTY Authentication Agent (kageant.exe)
...
'kitty' (0.74.4.11) was installed successfully!
```
Looks like kageant.exe is included in v0.74.4.11 again. No problems here.

**Comment by jrhawley** [comment]
Based on the issue being fixed in v0.74.4.11, I doubt that it's worth going back to the previous couple manifests to fix them. I might be worth just closing the issue
